test/unit/app/config/cron.test.js

```diff
@@ -0,0 +1,17 @@
+describe('cron config', () => {
+  const OLD_ENV = process.env
+
+  beforeEach(() => {
+    jest.resetModules() // Most important - it clears the cache
+    process.env = { ...OLD_ENV } // Make a copy
+  })
+
+  afterAll(() => {
+    process.env = OLD_ENV // Restore old environment
+  })
+
```

Comment:
just a note about this process, should have said it previously - did this on the portal ones, but it's advised against generally - as when you do parallel test runs you can get race conditions.  A better pattern is to use a getter in the actual config file, e.g. `getEnvVariable('OVERNIGHT_JOB_CRONTAB')`, then you can stub this.  Maybe one for next code freeze.  Didn't want to implement this though without talking about it.